verification/dependencies/flag/flag.gobra

```diff
@@ -18,10 +18,18 @@ package flag
 
 type Value interface {
 	pred Mem()
-	pred LowMem()
+
+	// Return whether the underlying data (or at least the data relevant for 
+	// 'String') is low.
+	// TODO: Once Gobra issue #846 is resolved, actually implement `IsLow`;
+	// at the moment, this can just be an abstract function.
```

Comment:
please make the same changes here as I asked for the first occurrence of this declaration in an interface